src/hevm/src/EVM.hs

```diff
@@ -1568,12 +1565,10 @@ unRefund n = do
     (filter (\(a,b) -> not (a == self && b == n)) refs)
 
 touchAccount :: Addr -> EVM()
-touchAccount a =
-  pushTo ((tx . substate) . touchedAccounts) a
+touchAccount = pushTo ((tx . substate) . touchedAccounts)
```

Comment:
my biggest problem with pointfree programming is that they missed the opportunity to call it pointless programming